packages/email-petition/src/email-petition.tsx

```diff
@@ -44,7 +43,7 @@ console.log("campaign: " + CAMPAIGN_NAME);
 
 const CAPTCHA_SITE_KEY = "6LdiglcpAAAAAM9XE_TNnAiZ22NR9nSRxHMOFn8E";
 
-export const Petition = () => {
+export const EmailPetition = (props: {defaultMessage: string, onSubmit?: () => void}) => {
```

Comment:
doesn't this need the petition slug (i.e. `duck`, `chicken`, etc.)? or am i missing something?

Reply:
oh yes. i got petition ID from env variables but forgot campaign name. i turned both into props.

hm someday i feel like we should only call the one backend service and it can call the email petition service internally.

Reply:
yeah this was def done in somewhat of a hurry. would def be ideal to only call 1 thing.